statsmodels/stats/tests/test_multi.py

```diff
@@ -348,3 +351,42 @@ def test_tukeyhsd():
     assert_almost_equal(confint, res[:, 1:3], decimal=2)
     assert_equal(reject, res[:, 3]<0.05)
 
+
+def test_local_fdr():
+
+    # Create a mixed population of Z-scores: 1000 standard normal and
+    # 20 uniformly distributed between 3 and 4.
+    grid = np.linspace(0.001, 0.999, 1000)
+    z0 = norm.ppf(grid)
+    z1 = np.linspace(3, 4, 20)
+    zs = np.concatenate((z0, z1))
+
+    # Exact local FDR for U(3, 4) component.
+    f1 = np.exp(-z1**2 / 2) / np.sqrt(2*np.pi)
+    r = len(z1) / float(len(z0) + len(z1))
+    f1 /= (1 - r) * f1 + r
+
+    fdr = local_fdr(zs)
+    fdr1 = fdr[len(z0):]
+
+    assert_allclose(f1, fdr1, rtol=0.05, atol=0.1)
+
+
+def test_empirical_null():
+
+    # Create a mixed population of Z-scores: 1000 standard normal and
+    # 20 uniformly distributed between 3 and 4.
+    grid = np.linspace(0.001, 0.999, 1000)
+    z0 = norm.ppf(grid)
+    z1 = np.linspace(3, 4, 20)
+    zs = np.concatenate((z0, z1))
+    emp_null = EmpiricalNull(zs, estimate_prob=True)
+
+    assert_allclose(emp_null.mean0, 0, atol=1e-5, rtol=1e-5)
+    assert_allclose(emp_null.scale0, 1, atol=1e-5, rtol=1e-2)
+    assert_allclose(emp_null.prob0, 0.98, atol=1e-5, rtol=1e-2)
+
+    # smoke tests
+    emp_null.pdf(0.)
+    emp_null.pdf(np.r_[-1, 0, 1])
```

Comment:
consistency instead of smoke test, something like
```
from scipy.stats import norm
assert_allequal(emp_null.pdf(np.r_[-1, 0, 1]), norm.pdf(np.r_[-1, 0, 1], loc=emp_null.mean0, scale=emp_null.scale0, rtol=1e-13)
```